external-crates/move/crates/move-analyzer/tests/ide_testsuite.rs

```diff
@@ -189,13 +189,25 @@ impl CompletionTest {
             line: lsp_use_line,
             character: lsp_use_col,
         };
-        let items = compute_completions(
-            symbols,
-            ide_files_root,
-            pkg_dependencies,
-            use_file_path,
-            use_pos,
+
+        // symbols do not change for each test, so we can reuse the same symbols
+        // but we need to recompute the cursor each time
+        let cursor_path = use_file_path.to_path_buf();
+        let cursor_info = Some((&cursor_path, use_pos));
+        let mut symbols_computation_data = SymbolsComputationData::new();
+        let mut cursor_context = compute_symbols_pre_process(
+            &mut symbols_computation_data,
+            &compiled_pkg_info,
+            cursor_info,
         );
+        cursor_context = compute_symbols_parsed_program(
```

Comment:
Consider adding a comment that this skips typed computation for speed

Reply:
Thank you! Done!